### DISCUSSION TITLE
Stored properties cannot be marked unavailable with '@available'

### DISCUSSION THREAD

**Discussion by PriyankaNovo** [SwiftPackageIndex/SwiftPackageIndex-Server#2662] [Q&A]
I have updated my xcode to 15. am getting error like "Stored properties cannot be marked unavailable with '@available'".  every time build project getting an this error can you please help in this .

**Comment by daveverwer** [comment]
Hi @PriyankaNovo

Are you trying to build the [SwiftPackageIndex-Server](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server) project?
